README.md

```diff
@@ -1,17 +1,40 @@
-![SST](http://sst-simulator.org/img/sst-logo-small.png)
+![SST](http:g/sst-simulator.org/img/sst-logo-small.png)
```

Comment:
What is the `:g/` here - shouldn't this be `//`

Reply:
That apparently is me mistyping a find replace command 🤦‍♂️